scripts/exregional_make_lbcs.sh

```diff
@@ -363,7 +355,12 @@ fi
 #-----------------------------------------------------------------------
 #
 num_fhrs="${#EXTRN_MDL_FHRS[@]}"
-for (( i=0; i<${num_fhrs}; i++ )); do
+bcgrp10=${bcgrp#0}
+bcgrpnum10=${bcgrpnum#0}
+for (( ii=0; ii<${num_fhrs}; ii=ii+bcgrpnum10 )); do
+  i=$(( ii + bcgrp10 ))
+  if [ ${i} -lt ${num_fhrs} ]; then
+    echo " group ${bcgrp10} processes member ${i}"
```

Comment:
This is the part that necessitates workflow update. LBCS files are downloaded in parallel in RRFS.

Reply:
Please explain what you mean by LBCS being downloaded in parallel in RRFS. 

Where are they downloaded from and in what task? Should this task be run as a metatask over each forecast lead time, or is it a group of lead times?

Reply:
Sorry I meant to say that `make_lbcs` is now parallel not `get_lbcs`. This does seem to help it finish much quicker. The `make_lbcs` task is now a metatask with something like this:
```
  metatask_make_lbcs_mem#mem#:
    var:
      bcgrp: '{% for h in range(0, workflow.BOUNDARY_PROC_GROUP_NUM) %}{{ " %02d" % h  }}{% endfor %}'
    task_make_lbcs_#bcgrp#_mem#mem#:
      account: '&ACCOUNT;'
```
The default for `BOUNDARY_PROC_GROUP_NUM` is 1 but is set to 10 in the RRFS test case to make the task finish faster.